components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/BuildScanDataFields.java

```diff
@@ -81,4 +78,8 @@ private static String summaryTotal(BuildScanData data, String avoidanceOutcome,
     private static String formatDuration(Duration duration) {
         return duration == null ? NO_VALUE : format(duration);
     }
+
+    private static String formatSerializationFactor(BigDecimal serializationFactor) {
+        return serializationFactor == null ? NO_VALUE : serializationFactor.setScale(2, RoundingMode.HALF_UP).doubleValue() + "x";
```

Comment:
I was not sure if we also wanted to append the `x` here, rather than in the scripts. I went ahead and moved it so that more is being done in Java, but still wanted to ask.

Reply:
I don't feel strongly either way, thus moving to Java is ok.